build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -23,31 +19,21 @@
   </ItemGroup>
 
   <Target Name="Build"
-          DependsOnTargets="LogOutput;
-                            InvalidateNuGetCache;
-                            @(NuSpec->'%(LayoutTargetName)')"
-          Inputs="@(NuSpec);@(PackageAssets)"
-          Outputs="@(NuSpec->'%(NuPkgOutput)')">
-
-    <PropertyGroup>
-      <NuGetExeVersion>3.4.4</NuGetExeVersion>
-      <NuGetTool>$(RepositoryRootDirectory).nuget\$(NuGetExeVersion)\nuget.exe</NuGetTool>
-      <NuGetDownloadUrl>https://dist.nuget.org/win-x86-commandline/v$(NuGetExeVersion)/NuGet.exe</NuGetDownloadUrl>
-      <NuGetDownloadCommand>powershell -noprofile -nologo -command "(new-object System.Net.WebClient).DownloadFile('$(NugetDownloadURL)', '$(NuGetTool)')"</NuGetDownloadCommand>
-    </PropertyGroup>
-
+          DependsOnTargets="InvalidateNuGetCache" >
+
+    <ItemGroup>
+      <BinPlaceContent Include="$(MSBuildThisFileDirectory)Sdkproject.csproj" />
+      <BinPlaceContent Include="$(MSBuildThisFileDirectory)Microsoft.NET.Sdk.nuspec" />
+      <NuspecFile Include="$(InitialOutDir)Microsoft.NET.Sdk.nuspec" />
+    </ItemGroup>
+
+    <Copy 
+        SourceFiles="@(BinPlaceContent)"
+        DestinationFolder="$(InitialOutDir)"
+    />
+    <Exec Command="$(DotNetTool) restore $(InitialOutDir)" />
```

Comment:
I don't understand why we are copying this project to the output directory and restoring it. Seems wrong.